### ISSUE TITLE
Error: Failed to check for transaction receipt:

### ISSUE THREAD

**Issue by REPTILEHAUS** [ethereum/web3.js#1534]
getting this on every transaction attempt.. using geth 1.8.2 and tried 1.8.1 also and web3  ^1.0.0-beta.33

```
this.web3.eth.sendSignedTransaction('0x' + serializedTx.toString('hex')) 
.on('transactionHash',  (hash) => {
  console.log("hash " + hash)
})
.on('receipt', (receipt) => {
  console.log("receipt " + receipt)
})
.on('confirmation', (confirmationNumber, receipt) => { 
  console.log("confirmation " + confirmationNumber + receipt)
})
.on('error', (error) => { 
  console.log("error " + error)
})
```

```

Error: Failed to check for transaction receipt:
{}
Stack trace:
["./node_modules/web3-utils/src/index.js"]/_fireError@http://localhost:4200/vendor.bundle.js:238222:17
["./node_modules/web3-core-method/src/index.js"]/Method.prototype._confirmTransaction/checkConfirmation/<@http://localhost:4200/vendor.bundle.js:220187:17
["./node_modules/zone.js/dist/zone.js"]/</</ZoneDelegate.prototype.invoke@http://localhost:4200/polyfills.bundle.js:2373:17
forkInnerZoneWithAngularBehavior/zone._inner<.onInvoke@http://localhost:4200/vendor.bundle.js:58558:24
["./node_modules/zone.js/dist/zone.js"]/</</ZoneDelegate.prototype.invoke@http://localhost:4200/polyfills.bundle.js:2372:17
["./node_modules/zone.js/dist/zone.js"]/</</Zone.prototype.run@http://localhost:4200/polyfills.bundle.js:2123:24
scheduleResolveOrReject/<@http://localhost:4200/polyfills.bundle.js:2857:29
["./node_modules/zone.js/dist/zone.js"]/</</ZoneDelegate.prototype.invokeTask@http://localhost:4200/polyfills.bundle.js:2406:17
forkInnerZoneWithAngularBehavior/zone._inner<.onInvokeTask@http://localhost:4200/vendor.bundle.js:58549:24
["./node_modules/zone.js/dist/zone.js"]/</</ZoneDelegate.prototype.invokeTask@http://localhost:4200/polyfills.bundle.js:2405:17
["./node_modules/zone.js/dist/zone.js"]/</</Zone.prototype.runTask@http://localhost:4200/polyfills.bundle.js:2173:28
drainMicroTaskQueue@http://localhost:4200/polyfills.bundle.js:2580:25
["./node_modules/zone.js/dist/zone.js"]/</</ZoneTask.invokeTask@http://localhost:4200/polyfills.bundle.js:2485:21
ZoneTask/this.invoke@http://localhost:4200/polyfills.bundle.js:2470:28
timer@http://localhost:4200/polyfills.bundle.js:4039:17
  vendor.bundle.js:124146:22
	emit http://localhost:4200/vendor.bundle.js:124146:22
	["./node_modules/web3-utils/src/index.js"]/_fireError/< http://localhost:4200/vendor.bundle.js:238245:13
	["./node_modules/zone.js/dist/zone.js"]/</</ZoneDelegate.prototype.invokeTask http://localhost:4200/polyfills.bundle.js:2406:17
	forkInnerZoneWithAngularBehavior/zone._inner<.onInvokeTask http://localhost:4200/vendor.bundle.js:58549:24
	["./node_modules/zone.js/dist/zone.js"]/</</ZoneDelegate.prototype.invokeTask http://localhost:4200/polyfills.bundle.js:2405:17
	["./node_modules/zone.js/dist/zone.js"]/</</Zone.prototype.runTask http://localhost:4200/polyfills.bundle.js:2173:28
	["./node_modules/zone.js/dist/zone.js"]/</</ZoneTask.invokeTask http://localhost:4200/polyfills.bundle.js:2481:24
	ZoneTask/this.invoke http://localhost:4200/polyfills.bundle.js:2470:28
	timer
```

**Comment by kyoyadmoon** [comment]
I solved the problem by upgrade geth version of API server to v1.8.11.

**Comment by nivida** [comment]
This should be fixed with the PR #2000 and will be released with the version 1.0.0-beta.38.
